patcher/src/ts/sidebar/ServerSelect.tsx

```diff
@@ -62,6 +63,13 @@ class SelectServer extends React.Component<SelectServerProps, SelectServerState>
     )
   }
 
+  trySelectCharacter = (shardID:number) => {
+    if(!this.props.charactersState || !this.props.charactersState.characters || 0 == this.props.charactersState.characters.length) return;
```

Comment:
if (
=== not ==
https://github.com/airbnb/javascript#whitespace--around-keywords
https://github.com/airbnb/javascript#comparison--eqeqeq